.github/actions/docker-build-and-push/action.yaml

```diff
@@ -44,14 +44,11 @@ runs:
           tags+=("type=match,pattern=\d+.\d+")
         fi
 
-        # Workaround for multiline strings
-        # https://github.community/t/set-output-truncates-multiline-strings/16852
-        tags_multiline=$(printf "%s\n" "${tags[@]}")
-        tags_multiline="${tags_multiline//'%'/'%25'}"
-        tags_multiline="${tags_multiline//$'\n'/'%0A'}"
-        tags_multiline="${tags_multiline//$'\r'/'%0D'}"
-
-        echo ::set-output name=tags::$tags_multiline
+        # Output multiline strings: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings
```

Comment:
:thinking: interesting that we have to use randomly generated EOF tokens to deal with injection attacks.